specs/electra/beacon-chain.md

```diff
@@ -232,6 +386,35 @@ def process_deposit_receipt(state: BeaconState, deposit_receipt: DepositReceipt)
     )
 ```
 
+#### New `process_execution_layer_exit`
+
+*Note*: This function is new in Electra:EIP7002.
+
+```python
+def process_execution_layer_exit(state: BeaconState, execution_layer_exit: ExecutionLayerExit) -> None:
+    validator_pubkeys = [v.pubkey for v in state.validators]
+    validator_index = ValidatorIndex(validator_pubkeys.index(execution_layer_exit.validator_pubkey))
```

Comment:
should there be a check to see if pubkey is in pubkeys and silent return if not?

Reply:
yep! everything coming from the EL should assume to be untrusted and so we should validate it, and if something doesn't line up should just skip over processing this particular `execution_layer_exit`

python will error here, which following the spec semantics would imply the block is invalid, giving anyone a cheap way to dos the chain -- not what we want!

Reply:
https://github.com/ethereum/consensus-specs/pull/3660